Assets/MRTK/Tests/TestUtilities/TestUtilities.cs

```diff
@@ -340,25 +338,25 @@ public static void ShutdownMixedRealityToolkit()
         public static void AssertAboutEqual(Vector3 actual, Vector3 expected, string message, float tolerance = 0.01f)
         {
             var dist = (actual - expected).magnitude;
-            Debug.Assert(dist < tolerance, $"{message}, expected {expected.ToString("0.000")}, was {actual.ToString("0.000")}");
+            Debug.Assert(dist < tolerance, $"{message}, expected {expected:0.000}, was {actual:0.000}");
```

Comment:
Just want to confirm this syntax also works for Unity 2018 users.

Reply:
Yep! Just built in 2018: 
![image](https://user-images.githubusercontent.com/3580640/141506742-447bf090-62cc-4a17-8a6a-a2222e016ec8.png)